### PR TITLE
Add three ticdc solution docs

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9031
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-tidb-to-tidb.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    tiup --tag upstream playground --host 0.0.0.0 --db 1 --pd 1 --kv 1 --tiflash 0 --ticdc 1
+    # Create a downstream cluster
+    tiup --tag downstream playground --host 0.0.0.0 --db 1 --pd 1 --kv 1 --tiflash 0 --ticdc 1
+    # View cluster status
+    tiup status
+    ```
+
+2. Initialize data.
+
+    By default, test databases are created in the newly deployed clusters. Therefore, you can use sysbench to generate test data and simulate data in real scenarios.
```
Comment (hfxsd) [inline migrate-from-tidb-to-tidb.md:45]:
```suggestion
    By default, test databases are created in the newly deployed clusters. Therefore, you can use [sysbench](https://github.com/akopytov/sysbench#linux) to generate test data and simulate data in real scenarios.
```

File: migrate-from-tidb-to-tidb.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    The access link is as follows:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    s3://backup?access-key=minio&secret-access-key=miniostorage&endpoint=http://${HOST_IP}:6060&force-path-style=true
+    ```
+
+## Step 2. Migrate full data
+
```
Comment (hfxsd) [inline migrate-from-tidb-to-tidb.md:115]: 这句话漏翻了：搭建好测试环境后，开始迁移数据。如果之前没有迁移过数据，则需要先进行全量迁移，再根据需要迁移增量数据。

File: migrate-from-tidb-to-tidb.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+    | Destination  | Size      | BackupTS           | Queue Time          | Execution Time      |
+    +--------------+-----------+--------------------+---------------------+---------------------+
+    | s3://backup  | 10315858  | 431434141450371074 | 2022-02-25 20:03:59 | 2022-02-25 20:03:59 |
+    +--------------+-----------+--------------------+---------------------+---------------------+
+    1 row in set (41.85 sec)
+    ```
+
+4. (Optional) Check data.
+
+    You can use [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) to check data consistency between upstream and downstream data at a certain time. The preceding `BACKUP` output shows that the upstream cluster finishes backup at 431434047157698561. The preceding `RESTORE` output shows that the downstream finishes restoration at 431434141450371074.
```
Comment (hfxsd) [inline migrate-from-tidb-to-tidb.md:180]:
```suggestion
    You can use [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) to check data consistency between upstream and downstream at a certain time. The preceding `BACKUP` output shows that the upstream cluster finishes backup at 431434047157698561. The preceding `RESTORE` output shows that the downstream finishes restoration at 431434141450371074.
```

File: migrate-from-tidb-to-tidb.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+        source-instances = ["upstream"]
+        target-instance = "downstream"
+        target-check-tables = ["*.*"]
+    ```
+
+## Step 3. Migrate incremental data
+
+1. Deploy TiCDC.
+
+    After finishing full data migration, deploy and configure a TiCDC to replicate incremental data. In production environments, deploy TiCDC as instructed in [Deploy TiCDC](/ticdc/deploy-ticdc.md). In this document, a TiCDC node has been started upon the creation of the test clusters. Therefore, we skip the step of deploying TiCDC and proceed with changefeed configuration.
```
Comment (hfxsd) [inline migrate-from-tidb-to-tidb.md:220]:
```suggestion
    After finishing full data migration, deploy and configure a TiCDC to replicate incremental data. In production environments, deploy TiCDC as instructed in [Deploy TiCDC](/ticdc/deploy-ticdc.md). In this document, a TiCDC node has been started upon the creation of the test clusters. Therefore, you can skip the step of deploying TiCDC and proceed with changefeed configuration.
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+aliases: ['/docs/dev/incremental-replication-between-clusters/']
+---
+
+# Replicate data between primary and secondary clusters
+
+This document describes how to configure a TiDB primary (upstream) cluster and a TiDB or MySQL secondary (downstream) cluster, and replicate incremental data from the primary cluster to the secondary cluster. The process includes the following steps:
+
+- Configure a TiDB primary cluster and a TiDB or MySQL secondary cluster.
+- Replicate incremental data from the primary cluster to the secondary cluster.
+- Recover data consistently by using Redo log when the primary cluster is down.
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:13]:
```suggestion
1. Configure a TiDB primary cluster and a TiDB or MySQL secondary cluster.
2. Replicate incremental data from the primary cluster to the secondary cluster.
3. Recover data consistently by using Redo log when the primary cluster is down.
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+
+To replicate incremental data from a running  TiDB cluster to its secondary cluster, you can use Backup & Restore [BR](/br/backup-and-restore-tool.md) and [TiCDC](/ticdc/ticdc-overview.md).
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:15]:
```suggestion
To replicate incremental data from a running TiDB cluster to its secondary cluster, you can use Backup & Restore [BR](/br/backup-and-restore-tool.md) and [TiCDC](/ticdc/ticdc-overview.md).
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+    tiup --tag upstream playground --host 0.0.0.0 --db 1 --pd 1 --kv 1 --tiflash 0 --ticdc 1
+    # Create the downstream cluster on Node B
+    tiup --tag downstream playground --host 0.0.0.0 --db 1 --pd 1 --kv 1 --tiflash 0 --ticdc 0
+    # View cluster status
+    tiup status
+    ```
+
+2. Initialize data.
+
+    By default, test databases are created in the newly deployed clusters. Therefore, you can use sysbench to generate test data and simulate data in real scenarios.
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:42]:
```suggestion
    By default, test databases are created in the newly deployed clusters. Therefore, you can use [sysbench](https://github.com/akopytov/sysbench#linux) to generate test data and simulate data in real scenarios.
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    sysbench oltp_write_only --config-file=./tidb-config --tables=3 run
+    ```
+
+4. Prepare external storage.
+
+    In full data backup, both the upstream and downstream clusters need to access backup files. It is recommended that you use [External storage](/br/backup-and-restore-storages.md#external-storages) to store backup files. In this document, we use Minio to simulate an S3-compatible storage service.
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:79]:
```suggestion
    In full data backup, both the upstream and downstream clusters need to access backup files. It is recommended that you use [External storage](/br/backup-and-restore-storages.md#external-storages) to store backup files. In this example, you can use Minio to simulate an S3-compatible storage service.
```
把 we 都全文查找替换掉吧

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+    ```
+
+    The preceding command starts a minio server on one node to simulate S3 services. Parameters in the command are configured as follows:
+
+    - Endpoint: `http://${HOST_IP}:6060/`
+    - Access-key: `minio`
+    - Secret-access-key: `miniostorage`
+    - Bucket: `redo`
+
+    The access link is as follows:
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:104]:
```suggestion
    The link is as follows:
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 10]
```diff
@@ -0,0 +1,322 @@
...
+    | Destination          | Size     | BackupTS           | Queue Time          | Execution Time      |
+    +----------------------+----------+--------------------+---------------------+---------------------+
+    | local:///tmp/backup/ | 10315858 | 431434141450371074 | 2022-02-25 20:03:59 | 2022-02-25 20:03:59 |
+    +----------------------+----------+--------------------+---------------------+---------------------+
+    1 row in set (41.85 sec)
+    ```
+
+4. (Optional) Check data.
+
+    Use [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) to check data consistency between upstream and downstream data at a certain time. The preceding `BACKUP` output shows that the upstream cluster finishes backup at 431434047157698561. The preceding `RESTORE` output shows that the downstream finishes restoration at 431434141450371074.
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:178]:
```suggestion
    Use [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) to check data consistency between upstream and downstream at a certain time. The preceding `BACKUP` output shows that the upstream cluster finishes backup at 431434047157698561. The preceding `RESTORE` output shows that the downstream finishes restoration at 431434141450371074.
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 11]
```diff
@@ -0,0 +1,322 @@
...
+
+    Create a changefeed configuration file `changefeed.toml`.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    [consistent]
+    # Consistency level, eventual means enabling consistent replication
+    level = "eventual"
+    # Use S3 to store redo logs, other options are local and nfs
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:236]:
```suggestion
    # Use S3 to store redo logs. Other options are local and nfs.
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 12]
```diff
@@ -0,0 +1,322 @@
...
+    storage = "s3://redo?access-key=minio&secret-access-key=miniostorage&endpoint=http://172.16.6.125:6060&force-path-style=true"
+    ```
+
+    In the upstream cluster, run the following command to create a changefeed from the upstream to the downstream clusters:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cdc  cli changefeed --sink-url "mysql://root:@172.16.6.124:4000" --config ./changefeed.toml
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:245]:
```suggestion
    tiup cdc cli changefeed --sink-url "mysql://root:@172.16.6.124:4000" --config ./changefeed.toml
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 13]
```diff
@@ -0,0 +1,322 @@
...
+    | @@global.tidb_gc_enable |
+    +-------------------------+
+    |                       1 |
+    +-------------------------+
+    1 row in set (0.00 sec)
+    ```
+
+## Step 4. Simulate a disaster in the upstream cluster
+
+Create a disastrous event in the upstream cluster while it is running. For example, you can terminate the tiup playground process by running Ctrl+C.
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:276]:
```suggestion
Create a disastrous event in the upstream cluster while it is running. For example, you can terminate the tiup playground process by pressing Ctrl+C.
```

File: replicate-betwwen-primary-and-secondary-clusters.md [hunk 14]
```diff
@@ -0,0 +1,322 @@
...
+
+## Step 5. Use redo log to ensure data consistency
+
+Normally, TiCDC concurrently writes transactions to downstream to increase throughout. When a changefeed is interrupted unexpectedly, the downstream may not have the latest data as it is in the upstream. To address inconsistency, we run the following command to ensure that the downstream data is consistent with the upstream data.
```
Comment (hfxsd) [inline replicate-betwwen-primary-and-secondary-clusters.md:280]:
```suggestion
Normally, TiCDC concurrently writes transactions to downstream to increase throughout. When a changefeed is interrupted unexpectedly, the downstream may not have the latest data as it is in the upstream. To address inconsistency, run the following command to ensure that the downstream data is consistent with the upstream data.
```

File: replicate-betwwen-primary-and-secondary-clusters.md (hunk 8 above)
Comment (shichun-0415) [reply replicate-betwwen-primary-and-secondary-clusters.md:79]: > 把 we 都全文查找替换掉吧

这个文档是示例，也就是我们操作给用户看，感觉改成 we 好像不合适。

File: migrate-from-tidb-to-tidb.md [hunk 15]
```diff
@@ -0,0 +1,292 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    sysbench oltp_write_only --config-file=./tidb-config --tables=3 run
+    ```
+
+4. Prepare external storage.
+
+    In full data backup, both the upstream and downstream clusters need to access backup files. It is recommended that you use [External storage](/br/backup-and-restore-storages.md) to store backup files. In this document, we use Minio to simulate an S3-compatible storage service.
```
Comment (shichun-0415) [inline migrate-from-tidb-to-tidb.md:80]:
```suggestion
    In full data backup, both the upstream and downstream clusters need to access backup files. It is recommended that you use [External storage](/br/backup-and-restore-storages.md) to store backup files. In this document, Minio is used to simulate an S3-compatible storage service.
```

File: TOC.md [hunk 16]
```diff
@@ -49,7 +49,8 @@
     - [Migrate and Merge MySQL Shards of Large Datasets](/migrate-large-mysql-shards-to-tidb.md)
     - [Migrate from CSV Files](/migrate-from-csv-files-to-tidb.md)
     - [Migrate from SQL Files](/migrate-from-sql-files-to-tidb.md)
-    - [Replicate Incremental Data between TiDB Clusters](/incremental-replication-between-clusters.md)
+    - [Migrate from one TiDB cluster to another TiDB cluster](/migrate-from-tidb-to-tidb.md)
+    - [Replicate incremental data from TiDB to Kafka](/replicate-incremental-data-to-kafka.md)
```
Comment (3AceShowHand) [inline TOC.md:53]: no `incremental`.

**Review by 3AceShowHand — APPROVED** [review]

File: TOC.md [hunk 17]
```diff
@@ -49,7 +49,8 @@
     - [Migrate and Merge MySQL Shards of Large Datasets](/migrate-large-mysql-shards-to-tidb.md)
     - [Migrate from CSV Files](/migrate-from-csv-files-to-tidb.md)
     - [Migrate from SQL Files](/migrate-from-sql-files-to-tidb.md)
-    - [Replicate Incremental Data between TiDB Clusters](/incremental-replication-between-clusters.md)
+    - [Migrate from one TiDB cluster to another TiDB cluster](/migrate-from-tidb-to-tidb.md)
+    - [Replicate data from TiDB to Kafka](/replicate-incremental-data-to-kafka.md)
```
Comment (shichun-0415) [inline TOC.md:53]:
```suggestion
    - [Replicate data from TiDB to Kafka](/replicate-data-to-kafka.md)
```

**Review by sdojjy — APPROVED** [review]

**Review by leoppro — APPROVED** [review]
LGTM

File: TOC.md (hunk 16 above)
Comment (shichun-0415) [reply TOC.md]: fixed

**Review by qiancai — APPROVED** [review]

**Pull request merged**